phpBB/install/config.yml

```diff
@@ -0,0 +1,6 @@
+database:
```

Comment:
Hi Nico, 
First of all thanks for your comments and taking a look. Nico its just for testing there. I will move it as you say in a sample directory.